aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -1290,15 +1297,24 @@ Tensor& orgqr_out_info(const Tensor& input, const Tensor& tau, Tensor& result, T
   infos.fill_(0);
 
   auto n = input.size(-1);
-  result = orgqr_stub(result.device().type(), result, tau, infos, n);
+  result = orgqr_stub(result.device().type(), result, tau_, infos, n);
   return result;
 }
 
-Tensor& orgqr_out(const Tensor& input, const Tensor& tau, Tensor& result) {
+Tensor& householder_product_out(const Tensor& input, const Tensor& tau, Tensor& result) {
   TORCH_CHECK(input.dim() >= 2, "orgqr: input must have at least 2 dimensions.");
   TORCH_CHECK(input.size(-2) >= input.size(-1), "orgqr: input.shape[-2] must be greater than or equal to input.shape[-1]");
   TORCH_CHECK(input.size(-1) >= tau.size(-1), "orgqr: input.shape[-1] must be greater than or equal to tau.shape[-1]");
 
+  TORCH_CHECK(input.dim() - tau.dim() == 1, "orgqr: Expected tau to have one dimension less than input, but got tau.ndim equal to ",
```

Comment:
Wouldn't the names for these operations start with "householder_product:" now? Or maybe even "torch.linalg.householder_product():"?

Reply:
Yes, I should change that.